code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -419,42 +419,96 @@
 
 /mob/living/carbon/human/proc/shadekin_get_energy()
 	var/datum/species/shadekin/SK = species
+	var/datum/species/crew_shadekin/BESK = species
 
 	if(!istype(SK))
```

Comment:
wont work like that
this should work
```suggestion
	if(istype(SK))
		return SK.get_energy()
	if(istype(BESK))
		return BESK.get_energy()
	return false
```

and please dont use all caps variables. Defines are all caps, variables always start lower case
I know its not your doing but please adjust it while you are at it.

Reply:
Went with what was there, thus I went with all caps. Will change.